### ISSUE TITLE
Build failure on 5.13

### ISSUE THREAD

**Issue by cylindrical2002** [neo4j/neo4j#13390]
I discovered that when I clone this repository to my Ubuntu 20.04，and I operate `mvn clean install`

**Neo4j Version:** 5.13 (default branch on 2024-1-25)
**Operating System:** Ubuntu 20.04
**Maven Version:** 3.9.6 with OpenJDK 17 

### Steps to reproduce
1. clone the repository
2. operate `mvn clean install` and wait until it reaches the 96 `Neo4j-Community

### Expected behavior
It would be a success

### Actual behavior
In `community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-39-54-803_4a641/conf/user-logs.xml`
```
<Configuration></Cunfigoratzion>
```
Obviously the spelling is wrong, but I don't know why this file is compiled

Here is the output:
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.neo4j.graphdb.factory.DatabaseManagementServiceBuilderIT
[INFO] Running org.neo4j.server.startup.ServerConsoleCommandIT
[INFO] Running org.neo4j.graphdb.facade.DatabaseManagementServiceFactoryIT
[INFO] Running org.neo4j.server.startup.Neo4jAdminServerCommandIT
[INFO] Running org.neo4j.server.startup.Neo4jCommandIT
[INFO] Running org.neo4j.graphdb.facade.DatabaseManagementServiceFactoryIT$ConfigurationIT
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 12.74 s -- in org.neo4j.graphdb.factory.DatabaseManagementServiceBuilderIT
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 15.39 s -- in org.neo4j.graphdb.facade.DatabaseManagementServiceFactoryIT$ConfigurationIT
[INFO] Running org.neo4j.graphdb.facade.DatabaseManagementServiceFactoryIT$ManagementServiceIT
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 16.33 s -- in org.neo4j.server.startup.ServerConsoleCommandIT
[ERROR] Tests run: 7, Failures: 2, Errors: 0, Skipped: 3, Time elapsed: 17.40 s <<< FAILURE! -- in org.neo4j.server.startup.Neo4jAdminServerCommandIT
[ERROR] org.neo4j.server.startup.Neo4jAdminServerCommandIT.startShouldFailOnInvalidUserLog4jConfig -- Time elapsed: 2.774 s <<< FAILURE!
java.lang.AssertionError: 

Expecting actual:
  "Validating Neo4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_4a641/conf/neo4j.conf
No issues found.

Validating user Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_4a641/conf/user-logs.xml
1 issue found.
Error at 1:18: 元素类型 "Configuration" 必须由匹配的结束标记 "</Configuration>" 终止。

Validating server Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_4a641/conf/server-logs.xml
No issues found.

Configuration contains errors. This validation can be performed again using 'neo4j-admin server validate-config'.
Run with '--verbose' for a more detailed error message.
"
to contain:
  ["Error at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>".",
    "Configuration contains errors."]
but could not find:
  ["Error at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>"."]
 
	at org.neo4j.server.startup.ServerCommandIT.startShouldFailOnInvalidUserLog4jConfig(ServerCommandIT.java:67)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)

[ERROR] org.neo4j.server.startup.Neo4jAdminServerCommandIT.startShouldBeAllowedWithWarningsOnInvalidServerLog4jConfig -- Time elapsed: 2.971 s <<< FAILURE!
java.lang.AssertionError: 

Expecting actual:
  "Validating Neo4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_7e1d8/conf/neo4j.conf
No issues found.

Validating user Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_7e1d8/conf/user-logs.xml
No issues found.

Validating server Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_7e1d8/conf/server-logs.xml
1 issue found.
Warning at 1:18: 元素类型 "Configuration" 必须由匹配的结束标记 "</Configuration>" 终止。

"
to contain:
  "Warning at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>"." 
	at org.neo4j.server.startup.ServerCommandIT.startShouldBeAllowedWithWarningsOnInvalidServerLog4jConfig(ServerCommandIT.java:80)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)

[ERROR] Tests run: 7, Failures: 2, Errors: 0, Skipped: 3, Time elapsed: 17.36 s <<< FAILURE! -- in org.neo4j.server.startup.Neo4jCommandIT
[ERROR] org.neo4j.server.startup.Neo4jCommandIT.startShouldFailOnInvalidUserLog4jConfig -- Time elapsed: 2.645 s <<< FAILURE!
java.lang.AssertionError: 

Expecting actual:
  "Validating Neo4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_4a641/conf/neo4j.conf
No issues found.

Validating user Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_4a641/conf/user-logs.xml
1 issue found.
Error at 1:18: 元素类型 "Configuration" 必须由匹配的结束标记 "</Configuration>" 终止。

Validating server Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_4a641/conf/server-logs.xml
No issues found.

Configuration contains errors. This validation can be performed again using 'neo4j-admin server validate-config'.
Run with '--verbose' for a more detailed error message.
"
to contain:
  ["Error at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>".",
    "Configuration contains errors."]
but could not find:
  ["Error at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>"."]
 
	at org.neo4j.server.startup.ServerCommandIT.startShouldFailOnInvalidUserLog4jConfig(ServerCommandIT.java:67)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)

[ERROR] org.neo4j.server.startup.Neo4jCommandIT.startShouldBeAllowedWithWarningsOnInvalidServerLog4jConfig -- Time elapsed: 2.980 s <<< FAILURE!
java.lang.AssertionError: 

Expecting actual:
  "Validating Neo4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_7e1d8/conf/neo4j.conf
No issues found.

Validating user Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_7e1d8/conf/user-logs.xml
No issues found.

Validating server Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_7e1d8/conf/server-logs.xml
1 issue found.
Warning at 1:18: 元素类型 "Configuration" 必须由匹配的结束标记 "</Configuration>" 终止。

"
to contain:
  "Warning at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>"." 
	at org.neo4j.server.startup.ServerCommandIT.startShouldBeAllowedWithWarningsOnInvalidServerLog4jConfig(ServerCommandIT.java:80)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)

[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.121 s -- in org.neo4j.graphdb.facade.DatabaseManagementServiceFactoryIT$ManagementServiceIT
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 19.67 s -- in org.neo4j.graphdb.facade.DatabaseManagementServiceFactoryIT
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   Neo4jAdminServerCommandIT>ServerCommandIT.startShouldBeAllowedWithWarningsOnInvalidServerLog4jConfig:80 
Expecting actual:
  "Validating Neo4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_7e1d8/conf/neo4j.conf
No issues found.

Validating user Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_7e1d8/conf/user-logs.xml
No issues found.

Validating server Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_7e1d8/conf/server-logs.xml
1 issue found.
Warning at 1:18: 元素类型 "Configuration" 必须由匹配的结束标记 "</Configuration>" 终止。

"
to contain:
  "Warning at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>"." 
[ERROR]   Neo4jAdminServerCommandIT>ServerCommandIT.startShouldFailOnInvalidUserLog4jConfig:67 
Expecting actual:
  "Validating Neo4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_4a641/conf/neo4j.conf
No issues found.

Validating user Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_4a641/conf/user-logs.xml
1 issue found.
Error at 1:18: 元素类型 "Configuration" 必须由匹配的结束标记 "</Configuration>" 终止。

Validating server Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jAdminServerCommandIT/25_06-12-22-519_4a641/conf/server-logs.xml
No issues found.

Configuration contains errors. This validation can be performed again using 'neo4j-admin server validate-config'.
Run with '--verbose' for a more detailed error message.
"
to contain:
  ["Error at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>".",
    "Configuration contains errors."]
but could not find:
  ["Error at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>"."]
 
[ERROR]   Neo4jCommandIT>ServerCommandIT.startShouldBeAllowedWithWarningsOnInvalidServerLog4jConfig:80 
Expecting actual:
  "Validating Neo4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_7e1d8/conf/neo4j.conf
No issues found.

Validating user Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_7e1d8/conf/user-logs.xml
No issues found.

Validating server Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_7e1d8/conf/server-logs.xml
1 issue found.
Warning at 1:18: 元素类型 "Configuration" 必须由匹配的结束标记 "</Configuration>" 终止。

"
to contain:
  "Warning at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>"." 
[ERROR]   Neo4jCommandIT>ServerCommandIT.startShouldFailOnInvalidUserLog4jConfig:67 
Expecting actual:
  "Validating Neo4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_4a641/conf/neo4j.conf
No issues found.

Validating user Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_4a641/conf/user-logs.xml
1 issue found.
Error at 1:18: 元素类型 "Configuration" 必须由匹配的结束标记 "</Configuration>" 终止。

Validating server Log4j configuration: /home/yue/Desktop/software/neo4j/community/neo4j/target/test data/org.neo4j.serv~.star~.Neo4jCommandIT/25_06-12-22-621_4a641/conf/server-logs.xml
No issues found.

Configuration contains errors. This validation can be performed again using 'neo4j-admin server validate-config'.
Run with '--verbose' for a more detailed error message.
"
to contain:
  ["Error at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>".",
    "Configuration contains errors."]
but could not find:
  ["Error at 1:18: The element type "Configuration" must be terminated by the matching end-tag "</Configuration>"."]
 
[INFO] 
[ERROR] Tests run: 28, Failures: 4, Errors: 0, Skipped: 6
[INFO] 
[INFO] 
[INFO] --- failsafe:3.1.2:verify (verify) @ neo4j ---
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Neo4j - Parent 5.13.0-SNAPSHOT:
[INFO] 
[INFO] Neo4j - Build resources ............................ SUCCESS [  0.679 s]
[INFO] Neo4j - Parent ..................................... SUCCESS [  0.935 s]
[INFO] Neo4j - Annotation processors ...................... SUCCESS [  6.093 s]
[INFO] Zstandard .......................................... SUCCESS [  0.078 s]
[INFO] Neo4j - Common ..................................... SUCCESS [ 12.573 s]
[INFO] Neo4j - Unsafe Access .............................. SUCCESS [  1.823 s]
[INFO] Neo4j - Native ..................................... SUCCESS [  1.369 s]
[INFO] Neo4j - Concurrent ................................. SUCCESS [  4.393 s]
[INFO] Neo4j - IO ......................................... SUCCESS [  9.733 s]
[INFO] Neo4j - Token API .................................. SUCCESS [  2.763 s]
[INFO] Neo4j - Resource interface ......................... SUCCESS [  0.235 s]
[INFO] Neo4j - Collections ................................ SUCCESS [  9.460 s]
[INFO] Neo4j - Graph Database API ......................... SUCCESS [  2.938 s]
[INFO] Neo4j - Exceptions ................................. SUCCESS [  1.608 s]
[INFO] Neo4j - Testing Utils Parent ....................... SUCCESS [  0.019 s]
[INFO] Neo4j - Test utilities ............................. SUCCESS [  8.746 s]
[INFO] Neo4j - Value ...................................... SUCCESS [ 10.226 s]
[INFO] Neo4j - Random Value ............................... SUCCESS [  5.153 s]
[INFO] Neo4j - Test IO utilities .......................... SUCCESS [  8.143 s]
[INFO] Neo4j - Logging .................................... SUCCESS [ 17.607 s]
[INFO] Neo4j - Test logging utilities ..................... SUCCESS [  0.428 s]
[INFO] Neo4j - Configuration .............................. SUCCESS [ 12.472 s]
[INFO] Neo4j - Locking .................................... SUCCESS [ 23.072 s]
[INFO] Neo4j - Schema ..................................... SUCCESS [  3.858 s]
[INFO] Neo4j - Procedure API .............................. SUCCESS [  0.337 s]
[INFO] Neo4j - Layout ..................................... SUCCESS [  0.335 s]
[INFO] Neo4j - Monitoring ................................. SUCCESS [  2.842 s]
[INFO] Neo4j - Diagnostics ................................ SUCCESS [  0.227 s]
[INFO] Neo4j - Native index ............................... SUCCESS [  1.253 s]
[INFO] Neo4j - gbptree-tests .............................. SUCCESS [01:49 min]
[INFO] Neo4j - ID Generator ............................... SUCCESS [ 36.948 s]
[INFO] Neo4j - Kernel API ................................. SUCCESS [ 13.461 s]
[INFO] Neo4j - CSV reading and parsing .................... SUCCESS [  7.072 s]
[INFO] Neo4j - Storage Engine Implementation utilities .... SUCCESS [ 12.199 s]
[INFO] Neo4j - Import Utilities ........................... SUCCESS [ 40.129 s]
[INFO] Neo4j - Storage Engine Test Utilities .............. SUCCESS [  0.254 s]
[INFO] Neo4j - Write-Ahead Log ............................ SUCCESS [  0.450 s]
[INFO] Neo4j - Write-Ahead Logging Test Utilities ......... SUCCESS [  0.266 s]
[INFO] Neo4j - Test Batch Import utilities ................ SUCCESS [  0.312 s]
[INFO] Neo4j - Database Layout API utilities .............. SUCCESS [  1.472 s]
[INFO] Neo4j - Test kernel API utilities .................. SUCCESS [  0.309 s]
[INFO] Neo4j - Native Spatial Index ....................... SUCCESS [ 20.013 s]
[INFO] Neo4j - Lucene Index ............................... SUCCESS [ 33.129 s]
[INFO] Neo4j - Fulltext index ............................. SUCCESS [  1.909 s]
[INFO] Neo4j - Graph Database Kernel ...................... SUCCESS [  6.848 s]
[INFO] Neo4j - Graph Database Kernel Test Utils ........... SUCCESS [  3.650 s]
[INFO] Neo4j - Record storage engine ...................... SUCCESS [01:26 min]
[INFO] Neo4j - Command Line ............................... SUCCESS [  1.535 s]
[INFO] Neo4j - Database Management System ................. SUCCESS [ 26.664 s]
[INFO] Neo4j - Community Cypher Build ..................... SUCCESS [  0.157 s]
[INFO] openCypher Front End Parent ........................ SUCCESS [  0.016 s]
[INFO] openCypher Test Utils .............................. SUCCESS [ 10.682 s]
[INFO] openCypher Utils ................................... SUCCESS [ 29.490 s]
[INFO] Neo4j - Cypher Configuration ....................... SUCCESS [ 15.329 s]
[INFO] Neo4j - Code Generator ............................. SUCCESS [ 36.715 s]
[INFO] openCypher Expressions ............................. SUCCESS [ 24.620 s]
[INFO] openCypher AST for the Cypher Query Language ....... SUCCESS [ 52.018 s]
[INFO] Neo4j - Cypher AST factory ......................... SUCCESS [  0.113 s]
[INFO] Neo4j - Cypher JavaCC based parser ................. SUCCESS [  3.025 s]
[INFO] Neo4j - Cypher ANTLR4 based parser ................. SUCCESS [  2.443 s]
[INFO] Neo4j - Neo4j CST factory .......................... SUCCESS [  6.896 s]
[INFO] Neo4j - Neo4j/OpenCypher AST factory ............... SUCCESS [06:15 min]
[INFO] openCypher Rewriting ............................... SUCCESS [ 39.462 s]
[INFO] Neo4j - Cypher Macros .............................. SUCCESS [  7.329 s]
[INFO] Neo4j - Cypher AST generator ....................... SUCCESS [ 15.707 s]
[INFO] openCypher Front End ............................... SUCCESS [01:20 min]
[INFO] Neo4j - Cypher Intermediate Representation ......... SUCCESS [ 32.872 s]
[INFO] Neo4j - Cypher Logical Plans ....................... SUCCESS [01:33 min]
[INFO] Neo4j - Cypher Logical Plan Builder ................ SUCCESS [01:02 min]
[INFO] Neo4j - Cypher Planner SPI ......................... SUCCESS [ 10.791 s]
[INFO] Neo4j - Cypher Runtime Utilities ................... SUCCESS [ 44.313 s]
[INFO] Neo4j - Graph Algorithms ........................... SUCCESS [  3.949 s]
[INFO] Neo4j - Cypher Interpreted Runtime ................. SUCCESS [02:02 min]
[INFO] Neo4j - Cypher Expression Evaluator ................ SUCCESS [ 13.481 s]
[INFO] Neo4j - Cypher Cache ............................... SUCCESS [  9.309 s]
[INFO] Neo4j - Cypher Graph Counts ........................ SUCCESS [ 11.338 s]
[INFO] Neo4j - Cypher Planner ............................. SUCCESS [04:52 min]
[INFO] Neo4j - Cypher Physical Planning ................... SUCCESS [ 39.406 s]
[INFO] Neo4j - Cypher Slotted Runtime ..................... SUCCESS [01:04 min]
[INFO] Neo4j - Cypher ..................................... SUCCESS [01:45 min]
[INFO] Neo4j - SSL ........................................ SUCCESS [ 14.804 s]
[INFO] Neo4j - Server API ................................. SUCCESS [  3.727 s]
[INFO] Neo4j - Security ................................... SUCCESS [ 13.704 s]
[INFO] Neo4j - Bolt Protocol Version 1 .................... SUCCESS [01:04 min]
[INFO] Neo4j - Fabric Parent .............................. SUCCESS [  0.025 s]
[INFO] Neo4j - Fabric ..................................... SUCCESS [ 41.657 s]
[INFO] Neo4j - Query Router ............................... SUCCESS [  0.625 s]
[INFO] Neo4j - Capabilities ............................... SUCCESS [  4.260 s]
[INFO] Neo4j - Community Procedures ....................... SUCCESS [ 29.563 s]
[INFO] Neo4j - Data Collector ............................. SUCCESS [  8.595 s]
[INFO] Neo4j - Consistency Checker ........................ SUCCESS [  1.846 s]
[INFO] Neo4j - Import Command Line Tool ................... SUCCESS [  5.741 s]
[INFO] Neo4j - Server ..................................... SUCCESS [ 37.768 s]
[INFO] Neo4j - SLF4J Provider ............................. SUCCESS [  2.881 s]
[INFO] Neo4j - Community .................................. FAILURE [01:21 min]
[INFO] Neo4j - Procedure Compiler Core .................... SKIPPED
[INFO] Neo4j - Community Integration Tests ................ SKIPPED
[INFO] Neo4j - Community Integration Test Support ......... SKIPPED
[INFO] neo4j-server-test-utils ............................ SKIPPED
[INFO] Neo4j - Test Harness ............................... SKIPPED
[INFO] Neo4j - Procedure Compiler Parent POM .............. SKIPPED
[INFO] Neo4j - Procedure Compiler Integration Tests ....... SKIPPED
[INFO] Neo4j - Graph Database Kernel Test ................. SKIPPED
[INFO] Neo4j - Community (Bill of Materials) .............. SKIPPED
[INFO] Neo4j - Record Storage Engine Integration Tests .... SKIPPED
[INFO] Neo4j - Community Kernel Integration Tests ......... SKIPPED
[INFO] Neo4j - Community Index Integration Tests .......... SKIPPED
[INFO] Neo4j - Community Import Integration Tests ......... SKIPPED
[INFO] Neo4j - Community DBMS Integration Tests ........... SKIPPED
[INFO] Neo4j - Community Graph Algorithms Integration Tests SKIPPED
[INFO] Neo4j - Community Bolt Integration Tests ........... SKIPPED
[INFO] Neo4j - Community Generic Integration Tests ........ SKIPPED
[INFO] Neo4j - Community Fabric Integration Tests ......... SKIPPED
[INFO] Neo4j - Community Security Integration Tests ....... SKIPPED
[INFO] Neo4j - Integration Tests without deep reflection into JDK internals SKIPPED
[INFO] Neo4j - SLF4J Provider Integration Tests ........... SKIPPED
[INFO] Neo4j - Cypher Testing ............................. SKIPPED
[INFO] Neo4j - Community Cypher Integration Tests ......... SKIPPED
[INFO] Neo4j - Test proxy ................................. SKIPPED
[INFO] Neo4j - Push To Cloud - Admin command .............. SKIPPED
[INFO] Neo4j - Boot ....................................... SKIPPED
[INFO] Neo4j - Cypher Specification Suite Tools ........... SKIPPED
[INFO] Neo4j - Cypher Compatibility Specification Suite ... SKIPPED
[INFO] Neo4j - Cypher Logical Plan Generator .............. SKIPPED
[INFO] Neo4j - Cypher Runtime Acceptance .................. SKIPPED
[INFO] Neo4j - Cypher Literal Interpreter ................. SKIPPED
[INFO] Neo4j - Cypher Shell Parent ........................ SKIPPED
[INFO] Neo4j - Cypher Shell ............................... SKIPPED
[INFO] Neo4j - Cypher Shell Integration Test .............. SKIPPED
[INFO] Neo4j - Cypher Shell Integration Test 2 ............ SKIPPED
[INFO] Neo4j - Community Build ............................ SKIPPED
[INFO] Neo4j - Server Assembler ........................... SKIPPED
[INFO] Neo4j - Community Server Assembler ................. SKIPPED
[INFO] Neo4j - Packaging Build ............................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  41:08 min
[INFO] Finished at: 2024-01-25T14:12:42+08:00
[INFO] ------------------------------------------------------------------------

```

**Comment by mnd999** [comment]
Looks like a locale issue, with that test failing on a non-English locale. The configuration file is intentionally incorrect, the test is verifying the behavior in the presence of invalid configuration, but it seems the error message is locale-sensitive.  The most straightforward thing to do is to run the tests with your JVM set to English.

**Comment by mnd999** [comment]
Fix will be in 5.17
